### DISCUSSION TITLE
[Bug Report] down 项目main分支之后运行报错

### DISCUSSION THREAD

**Discussion by qiaoqiao10001** [youzan/vant#12177] [Q&A]
### 重现链接

~

### Vant Cli 版本

~

### 描述一下你遇到的问题。

pnpm dev失败
<img width="988" alt="image" src="https://github.com/youzan/vant/assets/25581172/52aa5265-8bb1-4d60-ae88-d0d86fe6b10d">

### 重现步骤

1. 克隆仓库
2. pnpm install
3. pnpm run dev

**Comment by qiaoqiao10001** [comment]
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/zhangqiao/Downloads/vant/packages/vant-cli/lib/cli.js' imported from /Users/zhangqiao/Downloads/vant/packages/vant-cli/bin.js

**Comment by chenjiahan** [comment]
换成 Node 18 试一下，再执行下 pnpm run prepare
